src/fb.md

```diff
@@ -26,24 +26,33 @@ Before we go on, note that `planning = solution + tracking`, with solution playi
 
 ### Step II: Improve Your Solutions
 
-Next, you want to improve your solutions. There are several aspects you can work on, in no particular order. Not only are easier solutions faster to execute, but the ability to come up with them so will also help bring down your inspection time.
+Next, you want to improve your solutions. There are several aspects you can work on, in no particular order. Not only are easier solutions faster to execute, but the ability to come up with them will also help bring down your inspection time.
 
 - Influencing
     - `square + pair` and `line + line` are both two-stage solutions, and the point is we want to find alternative ways to solve the first stage i.e. `square`/`E-line` so we get a better second stage case i.e. `pair`/`D-line`, respectively. [^1]
+    - influencing involves prepending some move/s to your first stage solution so that the second stage case is better than without any influencing, and your overall FB solution is also better.
```

Comment:
I think the ones that aren't in `code blocks` should start with a capital letter instead